### PR TITLE
Added --no-submit-cell flag

### PR DESCRIPTION
**Pull request by scottjlee**
Added an option to skip generating a submission cell (`gen_submit_cell(...)`).

### CODE REVIEW

File: jassign/jassign.py [hunk 1]
```diff
@@ -3,20 +3,23 @@
 import argparse
 import pathlib
 
-from .to_ok import gen_views
+from to_ok import gen_views
```
Comment (papajohn) [inline jassign/jassign.py]: I think this change will make it impossible to run `jassign` as a command-line tool.

Comment (papajohn) [reply jassign/jassign.py]: If you can figure out a way to make it run both as a command-line tool and as a script, that would be great, but I didn't figure out how to make one import work for both.

Comment (scottjlee) [reply jassign/jassign.py]: We currently use it as a CLI tool and in a make file, so I'll just revert this back.

**Comment by scottjlee** [comment]
Does it automatically update on PyPI?

**Pull request merged**